doc/manual/src/command-ref/nix-copy-closure.md

```diff
@@ -14,62 +14,51 @@
 
 # Description
 
-`nix-copy-closure` gives you an easy and efficient way to exchange
-software between machines.  Given one or more Nix store _paths_ on the
-local machine, `nix-copy-closure` computes the closure of those paths
-(i.e. all their dependencies in the Nix store), and copies all paths
-in the closure to the remote machine via the `ssh` (Secure Shell)
-command.  With the `--from` option, the direction is reversed: the
-closure of _paths_ on a remote machine is copied to the Nix store on
-the local machine.
-
-This command is efficient because it only sends the store paths
-that are missing on the target machine.
-
-Since `nix-copy-closure` calls `ssh`, you may be asked to type in the
-appropriate password or passphrase.  In fact, you may be asked _twice_
-because `nix-copy-closure` currently connects twice to the remote
-machine, first to get the set of paths missing on the target machine,
-and second to send the dump of those paths.  When using public key
-authentication, you can avoid typing the passphrase with `ssh-agent`.
+Given _paths_ from one machine, `nix-copy-closure` computes the [closure](@docroot@/glossary.md#gloss-closure) of those paths (i.e. all their dependencies in the Nix store), and copies [store objects](@docroot@/glossary.md#gloss-store-object) in that closure to another machine via SSH.
+It doesn’t copy store objects that are already present on the other machine.
+
+> **Note**
+>
+> While the Nix store to use on the local machine can be specified on the command line with the [`--store`](@docroot@/command-ref/conf-file.md#conf-store) option, the Nix store to be accessed on the remote machine can only be [configured statically](@docroot@/command-ref/conf-file.md#configuration-file) on that remote machine.
+
+Since `nix-copy-closure` calls `ssh`, you may need to authenticate with the remote machine.
+In fact, you may be asked for authentication _twice_ because `nix-copy-closure` currently connects twice to the remote machine: first to get the set of paths missing on the target machine, and second to send the dump of those paths.
+When using public key authentication, you can avoid typing the passphrase with `ssh-agent`.
 
 # Options
 
-  - `--to`\
-    Copy the closure of _paths_ from the local Nix store to the Nix
-    store on _machine_. This is the default.
+  - `--to` _machine_
+
+    Copy the closure of _paths_ from a Nix store accessible from the local machine to the Nix store on the remote _machine_.
```

Comment:
This suggests that it doesn't support _to_ and _from_ simultaneously.
(Also it's not the flag that does the copying)
How about this?

```suggestion
    Defines the source of the paths to be from _machine_ instead of the local store or, if set, the default [store](@docroot@/command-ref/conf-file.md#conf-store).
```

Reply:
> This suggests that it doesn't support _to_ and _from_ simultaneously.

It doesn't, the direction is a boolean: https://github.com/NixOS/nix/blob/a57abbd143f8ed44e823c3244e93507f64020878/src/nix-copy-closure/nix-copy-closure.cc#L28-L31

Reply:
In fact, there is a problem with documentation because it's wrong! The code isn't pairing the flags with the _machine_ argument.

Reply:
Oh wow, this does change the whole picture.